packages/frontend-2/components/invite/dialog/CancelInvite.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <LayoutDialog v-model:open="isOpen" :buttons="dialogButtons" max-width="md">
+    <template #header>Cancel invite</template>
+    <p class="text-foreground text-body-xs">
+      Are you sure you want to cancel the invite for
+      <span class="font-medium">{{ email }}</span>
+      <span>?</span>
```

Comment:
This looks weird but else it rendered a space between the email and question mark because of the linter